src/main/webapp/WEB-INF/tags/student/comments/commentsForStudentsPanel.tag

```diff
@@ -0,0 +1,32 @@
+<%@ tag description="StudentComments - Comments for students" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/student/comments" prefix="comments" %>
+<%@ attribute name="commentRows" type="java.util.Collection" required="true" %>
+<div class="panel panel-primary">
+    <div class="panel-heading">
+        <strong>Comments for students</strong>
+    </div>
+    <div class="panel-body">
+        <c:forEach items="${commentRows}" var="commentRow" varStatus="i">
+            <c:set var="commentIdx" value="${i.index + 1}" />
+            <div class="panel panel-info student-record-comments ${commentRow.recipientDetails == 'you' ? 'giver_display-to-you' : 'giver_display-to-others'}">
```

Comment:
Maybe `giver_display-to-${commentRow.recipientDetails == 'you' ? 'you' : 'others'}` instead?
It's fine either way, though.

I'm more concerned over why it uses both `-` and `_`, but that's not related to this PR...